types/rand-token/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for rand-token 0.4
+// Project: https://github.com/sehrope/node-rand-token
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type SourceType = 'default' | 'math' | 'crypto';
+
+interface GeneratorOptions {
+    source: SourceType;
+    chars: string;
+}
+
+declare class RandToken {
```

Comment:
This isn't constructable, so don't make it a class. Just create an interface

```ts
export interface RandToken {
    generate(size: number, chars?: string[]): string;
}
```

and export each function at the top level

```ts
export function generator(generator: GeneratorOptions): RandToken;
export function uid(size: number): string;
```

Reply:
Cheers @DanielRosenwasser! I believe @sehrope wanted these types in the module repo itself, I'll create a PR on that repo updating them - I'll close this PR.

Good pickup with the constructable, not sure why I thought it was :confused: